runtime/proto/common.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+
+package org.corfudb.runtime;
+option java_package = "org.corfudb.runtime.proto";
+
+import "google/protobuf/any.proto";
+
+//TODO: Reasonable way to determine version?
+enum ProtocolVersion {
+  PROTOCOL_VERSION_INVALID = 0;
+  V0 = 1;
+}
+
+enum PriorityLevel {
+  PRIORITY_LEVEL_INVALID = 0;
+  NORMAL = 1;
+  HIGH = 2;
+}
```

Comment:
I noticed that when running `protoc`, the declaration of `NORMAL` and `HIGH` will be conflicted with those declared in `messages.proto`. 

Like `org.corfudb.runtime.HIGH already defined in common.proto`

 I think it's because `common.proto` and `messages.proto` are under the same package, although the generated Java source files belongs to different packages. 

Would this be a problem for us?